reference/configuration/debug.rst

```diff
@@ -16,6 +16,12 @@ key in your application configuration.
     # displays the actual config values used by your application
     $ php bin/console debug:config debug
 
+.. note::
+
+    The ``--resolve-env`` option is available with the ``debug:config``
```

Comment:
You should(must?) add a version added directive for that new option

Reply:
Done while merging. Thanks.